data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,133 @@ mission "Unfettered Jump Drive 3"
+			label meet
+			action
+				log "minor people" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. She fought you to test your resolve before you were given access to their weapons. You suspect she was holding back, or you would never have stood a chance.`
+			`	She does some kind of salute with her fist before leaving. "I most go now. I hope you will find the weapon you need, Captain."`
+			`	The engineer, who had stepped back during the fight, now proceeds to take you underground to what looks like a very basic weapon testing range. Many Hai are there, experimenting with small modifications, but never performing any extreme tests. Seeing your interest, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually fires before Unfettered captains can get their hands on it." She points to where a few captains watch the tests; apparently satisfied with what they see, one even requests a weapon to be taken to their ship. You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`
+			`	"Welcome to the special section of the outfitter, Captain! It is only for our most distinguished guests. Now that you have sold us multiple jump drives, I can tell you that we offer something much more valuable than money. For now, I can only offer you the mighty Tripulse Shredder, but eventually we will give you access to all kinds of weapons. Two Tripulse Shredders and our sympathy for one jump drive is the deal I propose," they say, presenting the weapons to you. Now that you can take a closer look, you notice these weapons seem fairly similar to other Hai weaponry, but enhanced in one way or another. There are even some weapons that appear to be ion-based, but one is smaller than an Ion Cannon whilst another one looks like a turret-mounted version of it.`
+			label weapons
+			choice
+				`	"Tell me more about those Tripulse Shredders I can buy."`
+					goto tripulse
+				`	"Can you tell me more about that small ionic weapon?"`
+				`	"I'd like to learn about the Ion Turret."`
+					goto turret
+				`	"I would just prefer the money, actually."`
+					goto money
+			scene "outfit/hai ionic blaster"
+			`	"Ah, the Blaster. It is favored by many Shield Beetle captains, because it can easily help them turn the tide of battle without needing to dedicate too much weapon space for it. Especially considering its other effects, its damage potential is serviceable. While it is true that it does not do enough ion damage to properly disable a ship on its own, it is able to jam their weapons and will give you the upper hand in a long fight."`
```

Comment:
```suggestion
			`	"Ah, the Blaster. It is favored by many Shield Beetle captains, because it can easily help them turn the tide of battle without needing to dedicate too much weapon space to it. Especially considering its other effects, its damage potential is serviceable. While it is true that it does not do enough ion damage to properly disable a ship on its own, it is able to jam their weapons and will give you the upper hand in a long fight."`
```